deployments/anvilprod/environment.py

```diff
@@ -56,7 +56,7 @@ def mkdict(previous_catalog: dict[str, str],
 
 
 anvil_sources = mkdict({}, 3, mkdelta([
-    mksrc('datarepo-dev-43738c90', 'ANVIL_1000G_2019_Dev_20230302_ANV5_202303032342', 22814),
+    mksrc('datarepo-dev-e53e74aa', 'ANVIL_1000G_2019_Dev_20230609_ANV5_202306121732', 6804),
```

Comment:
Does the drop in # of subgraphs make sense to you?

Reply:
A drop in the number of subgraphs was expected because many falsely-labeled supplementary files were inflating the subgraph count of the previous snapshot. I was a bit surprised that the drop was so large, but BigQuery confirms that they really did relabel that many files:

```
select is_supplementary, count(*)
from `datarepo-dev-e53e74aa.ANVIL_1000G_2019_Dev_20230609_ANV5_202306121732.anvil_file`
group by is_supplementary
```

Previous snapshot:
```
[{
  "is_supplementary": "false",
  "count": "9606"
}, {
  "is_supplementary": "true",
  "count": "16410"
}]
```

Updated snapshot:
```
[{
  "is_supplementary": "true",
  "count": "400"
}, {
  "is_supplementary": "false",
  "count": "25616"
}]
```

The total number of files remained constant at 26016.

Reply:
To summarize, yes, it does make sense to me.